sklearn/tests/test_base.py

```diff
@@ -232,3 +232,47 @@ def test_score_sample_weight():
                                    sample_weight=sample_weight),
                          msg="Unweighted and weighted scores "
                              "are unexpectedly equal")
+
+
+def test_clone_pandas_dataframe():
+    try:
+        import pandas as pd
+    except ImportError:
+        return
```

Comment:
I would rather raise a `SkipTest('pandas required')` exception to make it more explicit that this test was not run.